app/templates/main/catalogue.html

```diff
@@ -2,42 +2,111 @@
+              </ul>
+          
+          </div>
+          </div>
+          <div class="tna-column tna-column--width-2-3 tna-column--width-1-2-small tna-column--full-tiny">
+            <strong>Select a subject</strong>
+            <div class="subjects">
+              <ul class="subjects">
+                <li><a href="">Africa</a></li>
+                <li><a href="">Aid and development</a></li>
+                <li><a href="">Air force</a></li>
+                <li><a href="">Air transport</a></li>
+                <li><a href="">Ancient Monuments and Historical Buildings</a></li>
+                <li><a href="">Archives and libraries</a></li>
+                <li><a href="">Armed Forces (General Administration)</a></li>
+                <li><a href="">Army</a></li>
+                <li><a href="">Art, architecture and design</a></li>
+                <li><a href="">Asia</a></li>
+                <li><a href="">Australia and Pacific</a></li>
+              </ul>
+        
+            </div>
+        
+          </div>
+        
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+</div>
 
-{% block content %}
 <div class="tna-section">
   <div class="tna-container">
-    <div class="tna-column tna-column--width-2-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
-      <form action="{{ url('search:catalogue') }}" method="get">
-        <h1 class="tna-heading-xl">{{ pageTitle }}</h1>
-        {{ tnaSearchField({
-          'label': 'Search the catalogue',
-          'headingLevel': 2,
-          'headingSize': 'm',
-          'id': 'search',
-          'name': 'q',
-          'value': request.GET.get('q') or ''
-        }) }}
-      </form>
+    <div class="tna-column tna-column--width-3-3 tna-column--width-5-6-medium tna-column--full-small tna-column--full-tiny">
+
+      <div class="tna-error-summary" data-module="tna-error-summary" data-disable-autofocus="true">
+        <div role="alert">
+          <h2 class="tna-error-summary__heading tna-heading-m">
+            This site is still under development
+          </h2>
+          <p>Read our <a href="">blog post</a> for more information and help us out by providing <a href="">feedback</a>.</p>
+        </div>
+      </div>
```

Comment:
I don't think the error summary is the right component to use here - it's designed to summarise issues with form inputs.

If we could just use a plain heading and paragraph here for now, I can move the code I used for the notices in the main site somewhere where you can use them.

Reply:
I've adjusted the code to take out the error summary classes